executor/linux/step.go

```diff
@@ -91,13 +95,13 @@ func (c *client) PlanStep(ctx context.Context, ctn *pipeline.Container) error {
 	}
 
 	// update the step container environment
+	//
+	// https://pkg.go.dev/github.com/go-vela/pkg-executor/internal/step#Environment
 	err = step.Environment(ctn, c.build, c.repo, _step, c.Version)
 	if err != nil {
 		return err
 	}
 
-	_step.SetStatus(constants.StatusSuccess)
```

Comment:
was this an intentional removal?

Reply:
Yeah that was carry out from some of the past refactoring efforts that I missed submitting in those PRs 😞